App.tsx

```diff
@@ -1,19 +1,43 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import OnboardingScreen from './screens/Onboarding/OnboardingScreen';
+//import InformationScreen from './screens/Registration/InformationScreen/InformationScreen'
+//import InterestsScreen from './screens/Registration/InterestsScreen/InterestsScreen'
+
+import * as Font from 'expo-font';
+import React, { useState, useEffect,  } from 'react';
+
```

Comment:
Import the Splash Screen package.

```suggestion
import * as SplashScreen from 'expo-splash-screen'
```